rpc/gnmi/gnmi-specification.md

```diff
@@ -1054,6 +1054,11 @@ elements are omitted. In this scenario, `b` MUST be reverted to its default
 setting of True and the configuration of `c` MUST be deleted from the tree,
 and returned to its original un-configured setting.
 
+`replace` MUST not be used as a way to delete configuration by being supplied
```

Comment:
I suggest that we try and keep this a bit more specific.  Something like:

```
`replace` MUST NOT be used to delete the value at the path specified. For example...
```

The reason I think this is likely something that we should say is that if you have a data tree of `/a = {"one": 1, "two": 2}` and one sends `REPLACE /a` with a value of `{"one": 1}`, then technically one could argue that `/a/two` was deleted here.